hack/make/ubuntu

```diff
@@ -150,6 +158,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
@tianon I agree better documentation should be written, but AppArmor simply works out of the box. I haven't seen an Ubuntu machine where Docker and AppArmor weren't just working straight of the box. While documentation would be nice, it isn't necessary for this to be effective for users. It's a good, working default.

It can be argued that breaking out of a container is not something that users should care about, but to the degree that users *do* care about this, using an LSM is **essential**. With all kernels to date, it is trivial to break out of a container where an LSM is not used. Using an LSM cannot eliminate that threat, especially against kernel RCEs, but does makes it much more difficult to execute a local privilege escalation.

That said, I believe the policy could be locked down further, both by default, and by local preference. I have been developing better policies in order to improve the default, which I intend to introduce in a separate PR. The work I've done on building a hardened policy would be good material for documentation.